playbooks/ocim.yml

```diff
@@ -9,3 +9,9 @@
 
     - role: ocim
       tags: 'ocim'
+
+    - role: redis-exporter
+      tags: 'redis-exporter'
+      REDIS_EXPORTER_CHECK_KEYS: "{{ OCIM_QUEUE_REDIS_KEY }}"
+      REDIS_EXPORTER_REDIS_ADDR: "{{ OCIM_REDIS_ADDRESS }}"
```

Comment:
@0x29a, `OCIM_REDIS_ADDRESS` is missing in the role default variables file.

Reply:
@lgp171188, do you mean `playbooks/roles/redis-exporter/defaults/main.yml`? If so, it shouldn't be there since we're trying to make this role generic.